### PR TITLE
{ARO} minor CLI enhancements, including extension suppression

### PR DESCRIPTION
**Pull request by dkorzuno**
**Description**

The updates here are the latest development features to be added to Azure Red Hat OpenShift az cli client.

 - extension suppression
 - logging improvements
 - narrow permissions  

**Testing Guide**

None
These are the enhancements that we use for our Azure cli in the github.com/Azure/ARO-RP repository.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ARO

File: src/azure-cli/azure/cli/command_modules/aro/_help.py [hunk 1]
```diff
@@ -19,8 +19,6 @@
     text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet
   - name: Create a cluster with 5 compute nodes and Red Hat pull secret.
     text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet --worker-count 5 --pull-secret @pullsecret.txt
-  - name: Create a Private cluster
-    text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet --apiserver-visibility Private --ingress-visibility Private
```
Comment (jim-minter) [inline src/azure-cli/azure/cli/command_modules/aro/_help.py:23]: @dkorzuno this shouldn't be removed here, it is in fact missing and should be added in the ARO-RP copy of _help.py

File: src/azure-cli/azure/cli/command_modules/aro/custom.py [hunk 2]
```diff
@@ -66,18 +65,14 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
     if not client_sp:
         client_sp = aad.create_service_principal(client_id)
 
-    rp_client_id = FP_CLIENT_ID
-
+    rp_client_id = os.environ.get('AZURE_FP_CLIENT_ID', 'f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875')
```
Comment (jim-minter) [inline src/azure-cli/azure/cli/command_modules/aro/custom.py:68]: we should not read AZURE_FP_CLIENT_ID in this codebase, hard code to the uuid.  Probably keeping it as a constant, i.e. `FP_CLIENT_ID = 'f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875'`, is clearer

**Review by jim-minter — CHANGES_REQUESTED** [review]
minor changes requested

**Review by jim-minter — APPROVED** [review]
lgtm - azure-cli team please merge.

**Comment by jim-minter** [comment]
@fengzhou-msft bump

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**